pkg/console/subresource/consoleserver/config_builder_test.go

```diff
@@ -448,7 +453,7 @@ func TestConsoleServerCLIConfigBuilder(t *testing.T) {
 				},
 				Customization: Customization{
 					Perspectives: []Perspective{
-						{ID: "perspective1", Visibility: PerspectiveVisibility{State: PerspectiveAccessReview, AccessReview: &ResourceAttributesAccessReview{Required: []authorizationv1.ResourceAttributes{{Resource: "namespaces", Verb: "list"}}, Missing: []authorizationv1.ResourceAttributes{{Resource: "clusterroles", Verb: "list"}}}}},
+						{ID: "perspective1", Visibility: PerspectiveVisibility{State: PerspectiveAccessReview, AccessReview: &ResourceAttributesAccessReview{Required: []authorizationv1.ResourceAttributes{{Resource: "namespaces", Verb: "list"}}, Missing: []authorizationv1.ResourceAttributes{{Resource: "clusterroles", Verb: "list"}}}}, PinnedResources: []configv1.GroupVersionResource{{Group: "apps", Version: "v1", Resource: "Deployment"}, {Group: "core", Version: "v1", Resource: "ConfigMap"}}},
```

Comment:
Can we convert super long files like this to a multiline? The go formatter helps you when you adding a trailing ,